boards/arc/arduino_101_sss/arduino_101_sss.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: arduino_101_sss
+name: arduino_101_sss
```

Comment:
same thing though then, you'll have the same name on the board directory or the file name and two identifiers. Things will get confusing.

boards/arc/arduino_101_sss/arduino_101_sss.yaml
@@ -0,0 +1,12 @@
 +identifier: arduino_101_sss
 +name: arduino_101_sss

Four repetitions of the same name. I would default it like:
```python
with open(this_filename) as f:
  y = yaml.load(f.read())
if os.path.basename(this_filename) == "board.yaml":
  base_id = os.path.basename(os.path.dirname(this_filename))  # get the directory name
identifier = y.get('identifier', base_id)
name = y.get('name', identifier)
```
this way you can avoid repetition while at the same time allowing an override